src/pages/StartPage.js

```diff
@@ -4,25 +4,28 @@ import styled from 'styled-components/macro'
 
 export default function StartPage() {
   return (
-    <StyledContainer>
-      <h1>
-        Welcome <br /> to <br /> newAlumni!
-      </h1>
-      <img src={fishgroup} alt="fish" />
-    </StyledContainer>
+    <>
+      <StyledContainer>
+        <h1>Welcome to newAlumni!</h1>
+        <img src={fishgroup} alt="fish" />
+      </StyledContainer>
+    </>
   )
 }
 
 const StyledContainer = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  text-align: center;
+  height: 100vh;
```

Comment:
Why do you remove flexbox? Positioning the h1 and img with margins is not responsive.

Reply:
During the refactoring with Jan we changed the app grid to fix the "header issue" and ever since display flex on my StartPage didn't work anymore. I spent hours trying to fix it but I couldn't...

Thats the app grid:
`const AppGrid = styled.div`
  display: grid;
  grid-template-rows: 55px auto 55px;
  position: fixed;
  left: 0;
  right: 0;
  top: 0;
  bottom: 0;
  height: 100%;
``

If you have an idea, pls let me know!!

Reply:
Sorry, was busy with the hackathon. Did you fix it? I think `height: 100%` is not necessary when you 'nail' the corners of the container to the edges? But otherwise I have no idea at the moment..

Reply:
Sorry Jan, I unfortunately deleted the comment! I already found a solution. Thank you!